interfaces/edit_volunteer_cli.py

```diff
@@ -0,0 +1,79 @@
+from models.volunteer import Volunteer
+from models.camp import Camp
+from interfaces.admin_cli import ManageVolunteerMenu
+
+
+###---- Manage Volunteer Menu ----
+class EditVolunteerMenu(ManageVolunteerMenu):
+    def __init__(self):
+        super(EditVolunteerMenu, self).__init__()
+
+    def edit_volunteer_menu(self) -> None:
+        print("""
+         Select an item you'd like to edit:
+         --------
+         1) Firstname
+         2) Lastname
+         3) Phone number
+         4) Camp
+         5) Availability
+
+         R) Return to previous page
+         0) Log-out
+         """)
+
+    def precmd(self, option: str) -> int:
+        edit_volunteer_options = {"logout": 0,
+                                  "edit_firstname": 1,
+                                  "edit_lastname": 2,
+                                  "edit_phone": 3,
+                                  "edit_camp": 4,
+                                  "edit_availability": 5,
+                                  "return": "r"}
+
+        ### Add input error handling
+        if option.isdigit():
+            return list(edit_volunteer_options.keys())[list(edit_volunteer_options.values()).index(int(option))]
+        elif option == "r" or "R":
+            return list(edit_volunteer_options.keys())[list(edit_volunteer_options.values()).index(option.lower())]
+        else:
+            pass
```

Comment:
What should a user see in the else block if they fail to enter a valid value?

Reply:
Good question. I'll discuss with Vanessa to align it with our interface design.